src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  --header-height: 60px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  height: var(--header-height);
```

Comment:
Please try to use padding instead of hardcode height

Reply:
hight = content hight + paddings